content/geophysical_surveys/index.rst

```diff
@@ -15,23 +15,23 @@ Geophysical Surveys
 
 For each geophysical method, we will follow the framework:
 
-- Intro 
```

Comment:
this is possibly the file where the merge conflict is happening. Do you want to try and resolve it? We can sit together if that would be helpful